openbb_platform/providers/finviz/openbb_finviz/models/compare_groups.py

```diff
@@ -0,0 +1,295 @@
+"""Finviz Compare Groups Model."""
+# pylint: disable=unused-argument
+
+from typing import Any, Dict, List, Literal, Optional, Union
+
+from finvizfinance.group.overview import Overview
+from finvizfinance.group.performance import Performance
+from finvizfinance.group.valuation import Valuation
+from openbb_core.provider.abstract.data import ForceInt
+from openbb_core.provider.abstract.fetcher import Fetcher
+from openbb_core.provider.standard_models.compare_groups import (
+    CompareGroupsData,
+    CompareGroupsQueryParams,
+)
+from openbb_core.provider.utils.descriptions import DATA_DESCRIPTIONS
+from pandas import DataFrame
+from pydantic import Field, field_validator
+
+GROUPS = Literal[
```

Comment:
These literals should be inside `utils/literals.py`. The choice of file name is arbitrary.